lib/glimesh_web/live/homepage_live.html.leex

```diff
@@ -61,7 +61,7 @@
             <img src="/images/glimesh-verified.png" alt="" class="" style="max-height: 250px">
           </div>
           <div class="col-md-8 order-md-1">
-            <h2 class="featurette-heading"><%= gettext("Subscriptions for Everyone.") %> <br><small class="text-muted"><%= gettext("Level the playing field.") %></small></h2>
+            <h2 class="featurette-heading"><%= gettext("Level the playing field.") %> <br><small class="text-muted"><%= gettext("Subscriptions aren't just for the pros!") %></small></h2>
```

Comment:
Thought about this a lot, I don't like "Subscriptions aren't just for the pros!". It kinda insinuates that streamers without subscriptions are not professional on other platforms. 

What else can we choose for this sentence?

Reply:
I have to agree, another way of structuring and saying it would be: "Level the playing field, become your own professional streamer!" Or something like that, sounds corny but that's what I got for now.

Reply:
Yeah I agree on that one. What about just saying

We're all pros heres.